src/SqlStreamStore.Postgres/PostgresStreamStore.cs

```diff
@@ -257,7 +257,7 @@ public async Task<CheckSchemaResult> CheckSchema(CancellationToken cancellationT
         /// <returns>The database creation script.</returns>
         public string GetSchemaCreationScript()
         {
-            return _schema.Definition;
+            return _schema.Definition(_settings.Version);
         }
 
         /// <summary>
```

Comment:
Shouldn't this now not say `latest schema version 3.`

Reply:
This is to make a distinction between pgsql 9.6 and 14